docs/eks-installation.md

```diff
@@ -0,0 +1,26 @@
+# Deploying Antrea in AWS EKS
+
+This document describes steps to deploy Antrea in NetworkPolicy only mode to an AWS EKS cluster.
+
+Assuming you already have an EKS cluster, and have KUBECONFIG environment variable pointing to<br>
+the kubeconfig file of that cluster.
+
+Based on EKS worker Node MTU size and Kubernetes service cluster IP range, adjust
+**defaultMTU** and **serviceCIDR** values of antrea-agent.conf in ./build/yamls/antrea-eks.yml
+ accordingly, and apply
+./build/yamls/antrea-eks.yml to the EKS cluster.
+
+```bash
+kubectl apply -f ./build/yamls/antrea-eks.yaml 
```

Comment:
Probably just: kubectl apply -f antrea-eks.yml

Or if you like check the description at: https://github.com/vmware-tanzu/antrea/blob/master/docs/ipsec-tunnel.md

Reply:
done. good pointer. added instruction to download antrea-eks.yml.